cmd2/cmd2.py

```diff
@@ -4389,7 +4389,7 @@ def do_history(self, args: argparse.Namespace) -> Optional[bool]:
                     else:
                         fobj.write(f'{command.raw}\n')
             try:
-                self._run_editor(fname)
+                self.run_editor(fname)
```

Comment:
This seems like a reasonable change.  Once it is merged in, we should update the CHANGELOG and search through the Sphinx docs to see if there is anything we need to update there as well.